ansible/configs/base-infra/default_vars_ec2.yml

```diff
@@ -0,0 +1,117 @@
+---
+### AWS EC2 Environment settings
+
+ansible_user: ec2-user
+remote_user: ec2-user
+aws_region: us-east-2
+
+### Networking and DNS (AWS)
+
+subdomain_base_short: "{{ guid }}"
+
+subdomain_base: "{{ subdomain_base_short }}{{ subdomain_base_suffix }}"
+zone_internal_dns: "{{ guid }}.internal."
+chomped_zone_internal_dns: "{{ guid }}.internal"
+
+# Instance Configurations and Resources
+# 2023-02-21 tonykay@redhat.com Validated optimal sizing via AAP2 Controller docs
+# https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.1/html/red_hat_ansible_automation_platform_installation_guide/planning-installation?extIdCarryOver=true&sc_cid=701f2000001OH7YAAW
+
+bastion_instance_image: RHEL87GOLD-latest
```

Comment:
RHEL9 should be default in 2023.

Reply:
TBD, but ideally this is more an AgV thing. Customers are on 8 more than 9?